KuringApp/KuringApp/Department/DepartmentSelector.swift

```diff
@@ -52,28 +52,78 @@ struct DepartmentSelector: View {
     
     var body: some View {
         WithViewStore(self.store, observe: { $0 }) { viewStore in
-            List {
-                Section {
+            VStack {
+                HStack(alignment: .center, spacing: 10) {
+                    Text("대표 학과 선택")
+                        .font(.system(size: 18, weight: .bold))
+                        .foregroundStyle(Color.black)
+                }
+                .padding(.horizontal, 20)
+                .padding(.top, 24)
+                .padding(.bottom, 12)
+                .frame(width: 375, alignment: .leading)
+                
+                ScrollView {
+                    
                     ForEach(viewStore.addedDepartment) { department in
-                        Button {
-                            viewStore.send(.selectDepartment(id: department.id))
-                        } label: {
-                            Label(
-                                department.korName,
-                                systemImage: department == viewStore.currentDepartment
+                        HStack {
+                            Text(department.id)
+                                .font(.system(size: 16, weight: .medium))
+                                .foregroundColor(.black)
+                            Spacer()
+                            Image(
```

Comment:
```suggestion
                            Text(department.id)
                                .font(.system(size: 16, weight: .medium))
                                .foregroundColor(.black)
                                
                            Spacer()
                            
                            Image(
```
공백이 없다보니 modifier 랑 view 랑 구분이 잘 안되어서 뷰 간에는 vertical spacing 이 들어가면 좋겠습니다!

Reply:
반영했습니다